docs/samples/transformer/image_transformer/kfserving_sdk_transformer.ipynb

```diff
@@ -158,23 +162,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 41,
+   "execution_count": 12,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
-     "text": [
-      "[[-1.6099601984024048, -2.6461076736450195, 0.32844462990760803, 2.4825074672698975, 0.43524616956710815, 2.3108043670654297, 1.00056791305542, -0.4232763648033142, -0.5100948214530945, -1.7978394031524658]]\n",
-      "3\n"
-     ]
+     "text": "http://9.21.53.162/v1/models/cifar10:predict {'Host': 'cifar10.kubeflow.example.com'}\n[[-1.6099603176116943, -2.6461076736450195, 0.3284446597099304, 2.4825077056884766, 0.43524613976478577, 2.3108043670654297, 1.0005676746368408, -0.4232763648033142, -0.5100947022438049, -1.7978397607803345]]\n3\n"
     }
    ],
    "source": [
     "url = \"http://\" + cluster_ip + \"/v1/models/cifar10:predict\"\n",
-    "headers = { 'Host': 'cifar10-predict.kubeflow.example.com' }\n",
+    "headers = { 'Host': 'cifar10.kubeflow.example.com' }\n",
```

Comment:
can you help fix the namespace to not deploy to Kubeflow which is system namespace? thanks.

Reply:
Fixed